### DISCUSSION TITLE
Using fluent-bit on oracle linux

### DISCUSSION THREAD

**Discussion by Navaneeth-Bysani** [fluent/fluent-bit#5565] [Q&A]
I was trying to install fluent-bit on oracle linux using the following command.

`curl https://raw.githubusercontent.com/fluent/fluent-bit/master/install.sh | sh`

But every time I do that, I am getting the following error:

<img width="575" alt="Screenshot 2022-06-13 at 10 53 21 AM" src="https://user-images.githubusercontent.com/61292292/173285296-e3ba1d3f-47cf-48a1-ab8a-329e27100630.png">

It shows that fluent bit is not supported on oracle linux. 
Can someone help on how to use fluent bit on Oracle Linux?

**Comment by agup006** [comment]
@Navaneeth-Bysani I would recommend building from source on unsupported OSes

**Reply by patrick-stephens** [reply]
The install script does a quick sanity check of the OS so that is what is failing. As it is a RHEL compatible OS you may be able to just follow the manual install instructions to add the RPMs but as @agup006 says this is not an officially supported OS.
https://docs.fluentbit.io/manual/installation/linux/redhat-centos#configure-yum

**Comment by rghouzra** [comment]
i had same problem, i just build it from source code, you can use also docker image
